src/components/lyrics/Lyrics.jsx

```diff
@@ -3,14 +3,14 @@ import PropTypes from 'prop-types';
 
 const Lyrics = ({ lyrics }) => {
   return (
-    <div data-testid="lyrics">
+    <pre data-testid="lyrics">
       {lyrics.lyrics}
-    </div>
+    </pre>
   );
 };
 
 Lyrics.propTypes = {
-  lyrics: PropTypes.string.isRequired
+  lyrics: PropTypes.object.isRequired
 };
 
 export default Lyrics;
```

Comment:
No specifically this component, but where you are calling it. If you get 400 or a 404 in the response, it breaks the app. In the container where this is called you could add some logic in the useEffect to route back (location.goBack(-1)) or something similar. Great work, this code looks clean and neat!